gatsby-node.js

```diff
@@ -193,7 +193,7 @@ exports.createPages = async ({actions, graphql, reporter}) => {
 
     posts.forEach(post => {
       createPage({
-        path: post.node.frontmatter.permalink,
+        path: `/blog${post.node.frontmatter.permalink}`,
```

Comment:
I understand, that the operation would have to be: if a route comes from /blog/blabla it would have to redirect to /blabla without the /blog in the middle.
In this case, all the blogs have /blog, when it should not be like that.